### DISCUSSION TITLE
Window Grouping and Snapping

### DISCUSSION THREAD

**Discussion by Donar513** [f1multiviewer/issue-tracker#367] [Feature requests] [answered]
Allow/Create grouping and snapping of video and element windows to create seamless layouts or clusters of windows aligned and easily movable where needed.

**Comment by Hurri1Kane** [comment, marked as answer]
Hey, a user in the discord made a tutorial on how to do this
https://discord.com/channels/982375740764983366/1012990702793269328/1034209740579934220
